code-challenges.test.js

```diff
@@ -52,5 +85,50 @@ const studyMinutesWeek2 = {
+// That would give me a clue as to where to look to answer the question, it would broaden my thinking because I would be doing something new, and I would have a greater understanding of the methonds and for looping through an object. I really don't like using chatGPT. It's easy to get an answer. Very true, and if that's the purpose then okay. I do research what methods are used and how they work, but I just feel like I'm doing something wrong. If by chance you come back and say answer a different way, just assume I did not answer it because I will not other than what I am doing now.
 
 // Pseudo code:
+
+// function sortObjectValues(object) {
+//   // create a a simple function
+//   const values = [];
+//   // create a variable to hold the array
+//   for (let key in object) {
+//     // create a for loop that runs through an object checking each key. Once there are no more keys, it will stop.
+//     if (object.hasOwnProperty(key)) {
+//       // create a conditional that checks if the object does have a key
+//       // .hasOwnProperty checks to see if there is a value attached even if it is null or undefined. Returns false if the value is inherited or not declared.
+//       values.push(object[key]);
+//       // If it does, push.() the key value to values array
+//     }
+//   }
+//   values.sort((a, b) => a - b);
+//   // Use .sort() on values using a comparision function which compares two elements. If the value comes back negative, it tells .sort the a should come before b. If positive, then a is greater than b and should be placed behind. If it comes back as 0, it would mean they are the same value or equal to each other thus no change in order needed.
+
+//   return values;
+//   // Return the values array
+// }
+
+function sortObjectValues(obj) {
+  // create a simple function
+  return (Object.values(obj)).sort((a, b) => a - b);
+  // Use Object.valuse() to take in the object and place the keys(values) into an array
+  // Use .sort() on values using a comparision function which compares two elements. If the value comes back negative, it tells .sort the a should come before b. If positive, then a is greater than b and should be placed behind. If it comes back as 0, it would mean they are the same value or equal to each other thus no change in order needed. (I'm now writing that twice... whew)
+  // Return the value
+}
+
+// For whatever reason, ChatGPT used Array.from() on line 118. That seems redundant because all it would be able to do is iterate through since there is no arguement and just recreate the array. So I removed it.
+
+console.log(sortObjectValues(studyMinutesWeek2))
+// passed test
+
+// Now all this being said. If I answered question 2 the correct way using chatGPT, please tell me I am. Otherwise... I'm still at a lost.
```

Comment:
The purpose is to explore how to use AI as a developer. It is silly not to use a tool that can be very helpful. However, that doesn't mean AI gives you the "right" answer. Furthermore, if you are not taking the time to understand the output, you are limiting your learning. So we are getting practice in this area of tooling.

In this example, I would consider whether there is syntax that can be removed or reworked. But it looks like you did the work to understand the output and that is the point!